agent/pkg/dagent/utils/config.go

```diff
@@ -0,0 +1,12 @@
+package utils
```

Comment:
These utils duplicated in `dagent` and in `crane` it would be a great improvement to outsource in some type of common pkg. @nandor-magyar what do you think?

Reply:
it is "in-progress" (though being rather slow / iterative). A **better name** or **moving all utility functions** under /internal could be more aggressive step towards readability.